cluster/gce/upgrade.sh

```diff
@@ -102,34 +96,27 @@ function wait-for-master() {
 # $1 version string from command line
 function set_binary_version() {
```

Comment:
I should've caught this when it went in. These files are not specific to the GCE provider, but rather are Kubernetes release files, so this function can get lifted to the top-level script. When you do that, you should change the "gsutil cat" to a curl of the appropriate storage.googleapis.com location instead - that will mean any provider with a network connection can use them.